eng/liveBuilds.targets

```diff
@@ -222,4 +223,18 @@
   <PropertyGroup>
     <BundledRuntimeIdentifierGraphFile>$(RuntimeIdGraphDefinitionFile)</BundledRuntimeIdentifierGraphFile>
   </PropertyGroup>
+
+  <Target Name="RewriteRuntimePackDir"
+          Condition="'$(RunningPublish)' == 'true'"
+          DependsOnTargets="ResolveRuntimeFilesFromLocalBuild"
+          BeforeTargets="ResolveRuntimePackAssets">
+      <ItemGroup>
+        <!-- Remove AspNetCore runtime pack since we don't build it locally -->
+        <ResolvedRuntimePack Remove="Microsoft.AspNetCore.App.Runtime.$(RuntimeIdentifier)" />
+
+        <ResolvedRuntimePack Update="Microsoft.NETCore.App.Runtime.$(RuntimeIdentifier)">
+          <PackageDirectory>$(MicrosoftNetCoreAppRuntimePackDir)</PackageDirectory>
+        </ResolvedRuntimePack>
+      </ItemGroup>
+  </Target>
```

Comment:
This logic (picking the local runtime package directory when resolving the runtime pack) is already handled by targetingpack.targets which is used in many places in the repo. Can't you use that instead?

Reply:
I had trouble with this initially, but it may be easier now. Let me get to that in a follow-up PR.

Reply:
Which projects need this target in your change? Can we move that into a file that isn't imported by everything in the repository? MSBuild (target) evaluation isn't for free.

Reply:
Good point -- I think it's just crossgen2. I'll try to move it there.